src/CTA.WebForms2Blazor/ControlConverters/LabelControlConverter.cs

```diff
@@ -9,19 +11,28 @@ public class LabelControlConverter : ControlConverter
         protected override Dictionary<string, string> AttributeMap { 
             get
             {
-                throw new NotImplementedException();
+                return new Dictionary<string, string>()
+                {
+                    ["id"] = "for"
+                };
             } 
         }
-        protected override string BlazorName { get { throw new NotImplementedException(); } }
-        
-        public LabelControlConverter() : base()
-        {
-            
-        }
+        protected override string BlazorName { get { return "label"; } }
+
+        protected override string NodeTemplate { get { return @"{2}"; } }
 
         public override HtmlNode Convert2Blazor(HtmlNode node)
         {
-            throw new NotImplementedException();
+            var textAttr = node.Attributes.AttributesWithName("text").Single();
+            var labelText = textAttr.Value;
+
+            //Not sure if this needs to be handled and if this is expected behavior
+            if (node.Attributes.Contains("id"))
+            {
+                return Convert2BlazorFromParts(base.NodeTemplate, BlazorName, ConvertAttributes(node.Attributes), labelText);
+            }
+
+            return Convert2BlazorFromParts(NodeTemplate, null, null, labelText);
```

Comment:
I would suggest always porting `<asp:Label>` to a `<label>` element. This might differ from eShopOnBlazor, but it will be a safer implementation. Having a `<label>` element instead of plain text will provide more information to the front end and more structure to the file itself.